donate.html

```diff
@@ -595,47 +671,54 @@ <h4>Shipping Address</h4>
                 <input id="id_email" class="" type="text" name="email">
               </div>
             <!--Credit Card-->
-            {% comment %} Show the Credit Card Number Field only when the custom field for donation_only_paypal is {% endcomment %}
+            {% comment %} Show the Credit Card Fields only when the custom field for donation_only_paypal is {% endcomment %}
             {% if not page.custom_fields.donation_only_paypal or page.custom_fields.donation_only_paypal.strip|lower != 'y'  %}
-            <div id="ak-fieldbox-card_num" class="input-text ak-input-type-user cc-fields">
-              <label for="ak-card_num">{% filter ak_text:"field_card_num" %}Credit Card #{% endfilter %}</label>
-              <input type="hidden" name="required" value="card_num" id="ak-card_num-required">
-              <div id="ak-card_num-hosted" class="hosted-field"></div>
-              <input id="ak-card_num" type="text" name="card_num">
-            </div>
-            {% endif %}
+            <fieldset class="input-group cc-fields">
+              <div id="ak-fieldbox-card_num" class="input-text ak-input-type-user cc-fields">
+                <label for="ak-card_num">{% filter ak_text:"field_card_num" %}Credit Card #{% endfilter %}</label>
+                <input type="hidden" name="required" value="card_num" id="ak-card_num-required">
+                <div id="ak-card_num-hosted" class="hosted-field"></div>
+                <input id="ak-card_num" type="text" name="card_num">
+              </div>
+              <div class="inline_fields">
+                <div id="ak-fieldbox-exp_date" class="input-select input-cc-expiration ak-input-type-user cc-fields ak-err-below">
+                  <input type="hidden" name="required" value="exp_date_month" id="ak-exp_date_month-required">
+                  <label for="ak-exp_date_month">{% filter ak_text:"field_exp_date" %}Expiration Date{% endfilter %}</label>
+                  <div id="ak-exp_date-hosted" class="hosted-field"></div>
+                  <select id="ak-exp_date_month" class="input-cc-expiration-month c4 ct4 cm4 no-margin" type="text" name="exp_date_month">

```

Comment:
Potentially out of scope but what are your thoughts on moving this out to its own HTML file? This credit card section to be specific

Reply:
I think ideally, it would be good to reduce the size of that donate file, but it depends how reusable you want each section to be/how much it needs to be reusable.